tools/shell/src/scripts/migrate-groups-from-ow4.ts

```diff
@@ -0,0 +1,216 @@
+  console.log("---------------------")
+}
+
+async function dumpData() {
+  const PAGE_LIMIT = 20
+  async function dumpGroups() {
+    // Sanity configuration
+    const result = []
+    let page = 1
+    const getGroupUrl = (page: number) =>
+      `https://old.online.ntnu.no/api/v1/group/online-groups/?format=json&page=${page}`
+
+    while (true) {
+      const response = await fetch(getGroupUrl(page))
+      const data = await response.json()
+      if (data.next === null) {
+        break
+      }
+      page++
+      result.push(...data.results)
+
+      if (page > PAGE_LIMIT) {
+        console.error("Page limit reached in dumpGroups")
+        exit(1)
+      }
+    }
+
+    return result
+  }
+
+  async function dumpHobbys() {
+    const result = []
+    let page = 1
+    const getHobbyUrl = (page: number) => `https://old.online.ntnu.no/api/v1/hobbys/?format=json&page=${page}`
+
+    while (true) {
+      const response = await fetch(getHobbyUrl(page))
+      const data = await response.json()
+      if (data.next === null) {
+        break
+      }
+      page++
+      result.push(...data.results)
+
+      if (page > PAGE_LIMIT) {
+        console.error("Page limit reached in dumpHobbys")
+        exit(1)
+      }
+    }
+    return result
+  }
+
+  const groups = await dumpGroups()
+  const hobbies = await dumpHobbys()
+
+  const pathOfThisScript = import.meta.dirname
+  fs.writeFileSync(path.resolve(pathOfThisScript, "./groups.json"), JSON.stringify(groups, null, 2))
+  fs.writeFileSync(path.resolve(pathOfThisScript, "./hobbys.json"), JSON.stringify(hobbies, null, 2))
+}
+
+// @ts-ignore does not exist before running dumpData
+import groups from "./groups.json" assert { type: "json" }
+
+import { env } from "../env"
+// @ts-ignore does not exist before running dumpData
+import hobbies from "./hobbys.json" assert { type: "json" }
+
+async function insertDump() {
+  const committees = groups.filter((item) => item.group_type === "committee")
+  const nodeCommittees = groups.filter((item) => item.group_type === "node_committee")
+  const interestGroups = hobbies
+  const other = groups.filter((item) => item.group_type === "other")
+
+  console.log("\nCOMMITEES: ")
+  // biome-ignore lint/complexity/noForEach: <explanation>
+  committees.forEach((item) => console.log(item.name_short))
+
+  console.log("\nNODE COMMITTEES: ")
+  // biome-ignore lint/complexity/noForEach: <explanation>
+  nodeCommittees.forEach((item) => console.log(item.name_short))
+
+  console.log("\nOTHER: ")
+  // biome-ignore lint/complexity/noForEach: <explanation>
+  other.forEach((item) => console.log(item.name_short))
+
+  console.log("\nINTEREST GROUPS: ")
+  // biome-ignore lint/complexity/noForEach: <explanation>
+  interestGroups.forEach((item) => console.log(item.title))
+
+  console.log("\n\nInserting interest groups:")
+  for (const item of interestGroups) {
+    console.log(`Inserting ${item.title}`)
+    await prisma.interestGroup.create({
```

Comment:
Is there a reason we are doing direct prisma calls vs using the service layer? It's probably fine for these objects, but when it comes to things like events we have a lot of invariants we want to maintain